src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrUpdater.cs

```diff
@@ -420,7 +420,36 @@ public class VmrUpdater : VmrManagerBase, IVmrUpdater
             if (vmrPatchesToReapply.Any() && RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
                 cancellationToken.ThrowIfCancellationRequested();
-                await _localGitClient.CheckoutAsync(_vmrInfo.VmrPath, ".");
+                var result = await _localGitClient.RunGitCommandAsync(
```

Comment:
Maybe a link to the issue would be helpful here? As explanation into the logic behind this flow

Reply:
It's above in the parent block